study.md

```diff
@@ -31,7 +31,7 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+[23, 56, 3, 7].to_a.reverse
```

Comment:
This converts an array into an array and then returns a copy of the array with the elements sorted.  `[23, 56, 3, 7].sort!.reverse!`.